packages/sessions-sdk-react/src/calculate-notional.ts

```diff
@@ -0,0 +1,18 @@
+import * as dnum from "dnum";
+
+/**
+ * Calculate the notional (USD) value of a token amount
+ * @param amount - The token amount as a bigint
+ * @param decimals - The number of decimals for the token
+ * @param price - The price per token as a number
+ * @returns The product as a Dnum
+ */
+export const calculateNotional = (
+  amount: bigint,
+  decimals: number,
+  price: number,
+): dnum.Dnum => {
+  const priceDnum = dnum.from(price);
+  const amountDnum = [amount, decimals] as dnum.Dnum;
```

Comment:
I think you can just do `dnum.from([amount, decimals])` which I always prefer over the `as` keyword (the `as` keyword can typecast in dangerous ways)